SERVER/src/checkpoint_manager.py

```
@@ -1,105 +1,115 @@
+            # 1. Save the database
+            if os.path.exists(DB_PATH): # Use DB_PATH from config
+                shutil.copy(DB_PATH, os.path.join(checkpoint_dir, "dream_weaver.db"))
+
+            # 2. Save the server's LLM adapters
+            if os.path.exists(self.server_adapter_specific_path): # Use SERVER_ADAPTER_SPECIFIC_PATH
+                shutil.copytree(self.server_adapter_specific_path, os.path.join(checkpoint_dir, f"{self.server_Actor_id}_adapters"))
+            else:
+                print(f"Warning: Server LLM adapters not found at {self.server_adapter_specific_path}. Skipping adapter save.")
+
+            return f"Checkpoint '{checkpoint_name}' saved successfully.", self.list_checkpoints()
+        except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)